tests/page/page-click-scroll.spec.ts

```diff
@@ -79,9 +79,9 @@ it('should scroll into view display:contents with position', async ({ page, brow
   expect(await page.evaluate('window._clicked')).toBe(true);
 });
 
-it('should not crash when force-clicking hidden input', async ({ page, browserName }) => {
+it('should not crash when force-clicking hidden input', async ({ page, browserName, channel, browserMajorVersion }) => {
   it.info().annotations.push({ type: 'issue', description: 'https://github.com/microsoft/playwright/issues/18183' });
-  it.fixme(browserName === 'chromium');
+  it.skip(browserName === 'chromium' && !!channel && browserMajorVersion < 108);
```

Comment:
```suggestion
  it.skip(browserName === 'chromium' && browserMajorVersion < 108);
```
nit: I think we can remove that.